src/fides/api/ops/service/privacy_request/request_runner_service.py

```diff
@@ -415,6 +421,22 @@ async def run_privacy_request(
                 _log_exception(exc, CONFIG.dev_mode)
                 return
 
+        if policy.get_rules_for_action(
+            action_type=ActionType.consent
+        ) and can_run_checkpoint(
+            request_checkpoint=CurrentStep.consent,
+            from_checkpoint=resume_step,
+        ):
+
+            await run_consent_request(
+                privacy_request=privacy_request,
+                policy=policy,
+                graph=build_consent_dataset_graph(datasets),
+                connection_configs=connection_configs,
+                identity=identity_data,
+                session=session,
+            )
+
```

Comment:
Adding an additional step to the existing request runner, similar to how we have steps above for access and erasure.